dev/generator/mapstr.go

```diff
@@ -0,0 +1,287 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
@urso Found an "easy" solution. I copied over the mapstr file from libbeat and remove the methods I don't need. Not nice but works.

Reply:
Maybe create a tech-debt issue for us to know that might need to clean this up in the future.

Reply:
I filed https://github.com/elastic/beats/issues/14734